pyproject.toml

```diff
@@ -6,7 +6,7 @@
 name = "opentitan"
 version = "0.0.0"
 description = "Open-source hardware root-of-trust"
-requires-python = ">=3.8"
+requires-python = ">=3.8.1"
```

Comment:
Was this required for our deps to work, or did `3.8` have a security advisory?

Reply:
Some deps specify 3.8.1 as minimum